libqtile/widget/check_updates.py

```diff
@@ -63,8 +63,8 @@ class CheckUpdates(base.ThreadPoolText):
         ("no_update_string", "", "String to display if no updates available"),
     ]
 
-    def __init__(self, **config):
-        base.ThreadPoolText.__init__(self, "", **config)
+    def __init__(self, initial_text="", **config):
+        base.ThreadPoolText.__init__(self, initial_text, **config)
```

Comment:
If you do it this way, it won't be very clear from the docs that you're able to set `initial_text`.

An alternative would be to add `initial_text` to `defaults` and then do this:
```suggestion
    def __init__(self, **config):
        base.ThreadPoolText.__init__(self, config.pop("initial_text", ""), **config)
```